### ISSUE TITLE
Dependency addition to Joshua-Decoder/Joshua/pom.xml

### ISSUE THREAD

**Issue by chrismattmann** [joshua-decoder/joshua#239]
<dependency>     <groupId>args4j</groupId>     <artifactId>args4j</artifactId>     <version>2.32</version>    </dependency>
Joshua-Decoder/Joshua committer please add this dependency to pom.xml
thank you/
Martin

**Comment by chrismattmann** [comment]
from the Apache Incubator general@ discussion.

**Comment by lewismc** [comment]
@mjpost Joshua still builds with Ant right? Is there a specific requirement for the Maven pom.xml?

**Comment by mjpost** [comment]
Yep, it's ant-based. It's in the Ivy dependency file, `lib/ivy.xml`.

**Comment by lewismc** [comment]
Ack
So you want me to file a ticket to remove he Maven material?

On Friday, February 12, 2016, Matt Post <notifications@github.com> wrote:

> Yep, it's ant-based. It's in the Ivy dependency file, lib/ivy.xml.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joshua-decoder/joshua/issues/239#issuecomment-183468879>
> .
>


-- 
*Lewis*

**Comment by mjpost** [comment]
I don't think we need to remove it. Doesn't Maven replace Ivy? We could transition at some point.

**Comment by lewismc** [comment]
Oh, so you want to move away from Ant + Ivy to Maven?
If you want I can work on that one?

On Fri, Feb 12, 2016 at 1:08 PM, Matt Post <notifications@github.com> wrote:

> I don't think we need to remove it. Doesn't Maven replace Ivy? We could
> transition at some point.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joshua-decoder/joshua/issues/239#issuecomment-183488796>
> .
>



-- 
*Lewis*

**Comment by mjpost** [comment]
I wouldn't say I want to, but am definitely willing if someone wants to make the case for straight Maven over ant + ivy. In general, I prefer stability in build tools and am wary of jumping from ship to ship, but if there's a good case for it, I'd happily hear it. I've had some pressure to do so from another source, so maybe it'd be good to consider.

**Comment by lewismc** [comment]
I'll do it just now.

On Fri, Feb 12, 2016 at 7:03 PM, Matt Post <notifications@github.com> wrote:

> I wouldn't say I want to, but am definitely willing if someone wants to
> make the case for straight Maven over ant + ivy. In general, I prefer
> stability in build tools and am wary of jumping from ship to ship, but if
> there's a good case for it, I'd happily hear it. I've had some pressure to
> do so from another source, so maybe it'd be good to consider.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joshua-decoder/joshua/issues/239#issuecomment-183570893>
> .
>



-- 
*Lewis*

**Comment by lewismc** [comment]
This issue can be closed for the time being. it is not required as Maven is not the primary build system.
